tiny_dnn/core/kernels/conv2d_op_avx.h

```diff
@@ -21,12 +21,12 @@ namespace kernels {
 #ifdef CNN_USE_AVX
 
 // float ver
-template <typename Allocator>
+template <typename S1, typename S2, typename S3, typename S4>
```

Comment:
Do we really need to support different storage types for all tensor arguments?
gdb backtrace is hardly readable....